gadget/install/partition.go

```diff
@@ -56,11 +56,25 @@ func reloadPartitionTable(gadgetRoot string, device string) error {
 	}
 }
 
+type CreateOptions struct {
+	// The gadget root dir
+	GadgetRootDir string
+
+	// Create all missing partitions. If unset only
+	// role-{data,boot,save} partitions will get created and it's
+	// an error other partition is missing.
+	AllPartitions bool
```

Comment:
I don't see a test setting this to true

Reply:
I like the idea of making this clear - maybe something other than `unmanaged` though because neither role-* or others are unmanaged as such. Maybe `CreateAllMissingPartitions` ? Or  `AllowAllMissingPartitions` or `AllowNonRoleMissingPartitions` ? None of these is super nice but all are better than AllParititions so thanks for the suggestion :)

Reply:
Added a test and happy to rename (once we have settled on the name)

Reply:
My vote goes to `CreateAllMissingPartitions` :)